Adds INF2_ISTOGGLEABLE skill flag (#6737)
rathena/rathena: doc/skill_db.txt

IgnoreCicada				- Not blocked by SC_UTSUSEMI or SC_BUNSINJYUTSU (When TargetType is Weapon only).
ShowScale					- Shows AoE area while casting
IgnoreGtb					- Not blocked by Golden Thief Bug card.

> IsToggleable				- Skill is able to be toggled on and off. When toggled off the skill doesn't consume HP/SP.

Reviewer: ```suggestion
IsToggleable				- Skill can be toggled on and off. When toggled off the skill doesn't consume HP/SP.
```
